doc/src/manual/constructors.md

```diff
@@ -552,7 +552,8 @@ ERROR: MethodError: no method matching SummedArray(::Vector{Int32}, ::Int32)
 Closest candidates are:
   SummedArray(::Vector{T}) where T at none:4
 Stacktrace:
- [1] top-level scope at none:1
```

Comment:
Can we just change these to `[...]`.

Reply:
Should the
```
 Closest candidates are:
  SummedArray(::Vector{T}) where T at none:4 
```
still be included? But maybe also not that important.

Reply:
I think that can be kept.